engine/collection/message_hub/message_hub.go

```diff
@@ -450,6 +453,7 @@ func (h *MessageHub) Process(channel channels.Channel, originID flow.Identifier,
 // forwardToOwnVoteAggregator converts vote to generic `model.Vote`, logs vote and forwards it to own `voteAggregator`.
 // Per API convention, timeoutAggregator` is non-blocking, hence, this call returns quickly.
 func (h *MessageHub) forwardToOwnVoteAggregator(vote *messages.ClusterBlockVote, originID flow.Identifier) {
+	h.engineMetrics.MessageReceived(metrics.EngineCollectionMessageHub, metrics.MessageClusterBlockVote)
```

Comment:
```suggestion
```
I have a feeling that spanning the received and processed/dropped metrics over two engines might making understanding them more difficult. In all existing cases I'm aware of, these metrics are used within one engine/component. Using the same engine label also lets you do queries that aggregate all the message types for a particular engine, like `processed - received by (engine)`. 

Suggestion: move the `MessageReceived` line into `VoteAggregator` and `TimeoutAggregator`.

Reply:
I see your argument here, when I was designing this I was thinking about tracking by _object_ not as much as by _engine_. This is the reason why messages are reported in different places, it gives a strong understanding where the message was received, handled and dropped and by which engine. Of course we can argue what approach is more suitable but I would like to have a generalized common pattern of reporting metrics.